neps/nep-0514.md

```diff
@@ -0,0 +1,173 @@
+  them, and don't exercise the unique code paths.
+* Separate chains can't process real transaction traffic. The traffic must
+  either be synthetic or "inspired" by real traffic.
+* Each such test has a significant cost of running multiple nodes, in some
+  cases, tens of nodes.
+
+## Specification
+
+The proposal suggests altering the number of block producer seats to ensure that
+a portion of the `testnet` validating nodes become chunk-only producers.
+
+The desired `testnet` configuration is the following:
+
+* `"num_block_producer_seats": 20,`
+* `"num_block_producer_seats_per_shard": [ 20, 20, 20, 20 ],`
+* `"num_chunk_only_producer_seats": 100,`
+
+I suggest to implement the change for all networks that are not `mainnet` and
+have `use_production_config` in the genesis file. `use_production_config` is a
+sneaky parameter in `GenesisConfig` that lets protocol upgrades to change
+network's `GenesisConfig`.
+
+I don't have a solid argument for lowering the number of chunk producer seats,
+but that reflects the reality that we don't expect a lot of nodes joining
+`testnet`. It also makes it easier to test the case of too many validating nodes
+willing to join a network.
+
+## Reference Implementation
+
+[#9563](https://github.com/near/nearcore/pull/9563)
+
+If `use_production_config`, check whether `chain_id` is eligible, then change
+the configuration as specified above.
+
+## Security Implications
+
+The block production in `testnet` becomes more centralized. It's not a new
+concern as 50% of stake is already owned by nodes operated by the protocol
+developers.
+
+## Alternatives
+
+See above.
+
+## Future possibilities
+
+Adjust the number of block and chunk producer seats according to the development
+of the number of `testnet` validating nodes.
+
+## Consequences
+
+### Positive
+
+* Chunk-only production gets tested in `testnet`
+* Development of State Sync and other features related to chunk-only producers accelerates
+
+### Neutral
+
+* `testnet` block production becomes more centralized
+
+### Negative
+
+* Any?
+
+### Backwards Compatibility
+
+During the protocol upgrade, some nodes will become chunk-only producers.
+
+The piece of code that updates `testnet` configuration value will need to be
+kept in the database in case somebody wants to generate `EpochInfo` compatible
+with the protocol versions containing the implementation of this NEP.
+
+## Changelog
+
+### 1.0.0 - Initial Version
+
+> Placeholder for the context about when and who approved this NEP version.
+
+#### Benefits
+
+> List of benefits filled by the Subject Matter Experts while reviewing this version:
+
+* Benefit 1
+* Benefit 2
+
+#### Concerns
+
+> Template for Subject Matter Experts review for this version:
+> Status: New | Ongoing | Resolved
+
+|   # | Concern | Resolution | Status |
+| --: | :------ | :--------- | -----: |
+|   1 |         |            |        |
+|   2 |         |            |        |
```

Comment:
Given that Benefits and Concerns are already summarized in "Consequences" section, the benefits and concerns may be skipped here, but we still need to capture how the decision has been made. Here is an example:

```suggestion
### 1.0.0 - Initial Version

The Protocol Working Group members approved this NEP on MMM DD, 2023
```